webpages/settings/components/addon-body.html

```diff
@@ -263,12 +262,12 @@
   }
   .addon-message {
     padding: 8px;
-    border: 2px solid;
```

Comment:
Without the border the yellow notices are very hard to distinguish from the normal ones without the border. Could `--notice-background` be changed to yellow in dark mode?